banners/desktop/C24_WMDE_Desktop_DE_22/styles/styles.scss

```diff
@@ -12,7 +12,7 @@
 @use 'Banner';
 @use 'src/themes/UseOfFunds/UseOfFunds';
 @use 'MainBanner' with (
-	$banner-height: 357px,
+	$banner-height: 368.8px,
```

Comment:
Giving this a fractional value will cause issues as OSs/browsers do subpixel rendering differently, and it's considered to not be a best practice in CSS. I would recommend leaving it at 357px, and reducing the font size of the text in the message in both banners.

Reply:
You can actually look at the en banner for this, as that also has a load of text

Reply:
We had the slider break point increased at 1550 as suggested by the PM.